Gemfile

```diff
@@ -19,6 +19,8 @@ gem 'devise'
 
 gem 'pronto', '~> 0.9.5'
 gem 'pronto-rubocop', '~> 0.9.0'
+gem 'rest-client', '~> 2.0', '>= 2.0.2'
+gem 'ap'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `ap` should appear before `rest-client`.